acceptance/auth/bundle_and_profile/script

```diff
@@ -0,0 +1,22 @@
+title "Inside the bundle, no flags"
+trace errcode $CLI current-user me | jq .userName
+
+title "Inside the bundle, target flags"
+trace errcode $CLI current-user me -t dev | jq .userName
+
+title "Inside the bundle, target and matching profile"
+trace errcode $CLI current-user me -t dev -p DEFAULT | jq .userName
+
+title "Inside the bundle, profile flag not matching bundle host (TODO: this should pass with no error and use profile_name host)"
```

Comment:
Could you also use Badness field in test.toml to encode what's wrong / unexpected in this output?